crates/sui-types/src/execution_status.rs

```diff
@@ -171,6 +171,12 @@ pub enum ExecutionFailureStatus {
 
     #[error("Certificate is on the deny list")]
     CertificateDenied,
+
+    #[error(
+        "Sui Move Bytecode Verification Timeout. \
+        Please run the Sui Move Verifier for more information."
+    )]
+    SuiMoveVerificationTimedout,
```

Comment:
Bit strange since we should never see this in execution but of course there isn't much we can do here. 

Do you think it is worth it to add some debug_asserts or logging or something attesting to that we shouldn't see this error when metering is off?

Reply:
Correct the error placement is wrong 
But its the simplest way without changing the whole verifier error return. Or do you reckon we should do that? Maybe introduce a new error called SuiVerifierError?

Sure thing on the debug asserts.

Reply:
> Do you think it is worth it to add some debug_asserts or logging or something attesting to that we shouldn't see this error when metering is off?

Added debug assert for unmetered